be/src/olap/types.h

```diff
@@ -255,6 +255,36 @@ struct BaseFieldtypeTraits : public CppTypeTraits<field_type> {
     }
 };
 
+template <typename T>
+OLAPStatus convert_int_from_varchar(void* dest, const void* src) {
+    using SrcType = typename CppTypeTraits<OLAP_FIELD_TYPE_VARCHAR>::CppType;
+    auto src_value = reinterpret_cast<const SrcType*>(src);
+    StringParser::ParseResult parse_res;
+    T result = StringParser::string_to_int<T>(src_value->get_data(), src_value->get_size(), &parse_res);
+    if (UNLIKELY(parse_res != StringParser::PARSE_SUCCESS)) {
+        return OLAPStatus::OLAP_ERR_INVALID_SCHEMA;
+    }
+    if (typeid(T) == typeid(CppTypeTraits<OLAP_FIELD_TYPE_LARGEINT>::CppType)) {
```

Comment:
I think typeid is a runtime operation whose performance may be bad.
I think there are two choices for you 
1. write a specialization for largeint
2. use memcpy for all the type

Reply:
> I think typeid is a runtime operation whose performance may be bad.
> I think there are two choices for you
> 
> 1. write a specialization for largeint
> 2. use memcpy for all the type

done